src/functions-and-arrays.js

```diff
@@ -78,7 +151,16 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(wordsCount, word) {
+
+  let timesWord = 0;
+  for(let i=0; i<wordsCount.length; i++){
+    if(wordsCount[i] === word){
+      timesWord++;
+    } else{}
```

Comment:
Here it would be better to remove the `else`

```suggestion
    } 
```